api/src/test/java/org/openmrs/module/mksreports/reports/OutpatientConsultationReportManagerTest.java

```diff
@@ -89,49 +92,59 @@ public void testReport() throws Exception {
-			assertNotNull(total1);
-			assertEquals(1, total1.getSize());
-			assertTrue(total1.getMemberIds().contains(6));
-			Cohort total2 = (Cohort) row.getColumnValue("MALARIA." + OutpatientConsultationReportManager.col18);
-			assertNotNull(total2);
-			assertEquals(0, total2.getSize());
-			Cohort total3 = (Cohort) row.getColumnValue("FEVER." + OutpatientConsultationReportManager.col17);
-			assertNotNull(total3);
-			assertEquals(1, total3.getSize());
-			Cohort total4 = (Cohort) row.getColumnValue("FEVER." + OutpatientConsultationReportManager.col18);
-			assertNotNull(total4);
-			assertEquals(0, total4.getSize());
+			Cohort allMalesWithMalaria = (Cohort) row.getColumnValue("MALARIA." + OutpatientConsultationReportManager.col17);
+			assertNotNull(allMalesWithMalaria);
+			assertEquals(1, allMalesWithMalaria.getSize());
+			assertTrue(allMalesWithMalaria.getMemberIds().contains(6));
+			Cohort allFemalesWithMalaria = (Cohort) row
+			        .getColumnValue("MALARIA." + OutpatientConsultationReportManager.col18);
+			assertNotNull(allFemalesWithMalaria);
+			assertEquals(1, allFemalesWithMalaria.getSize());
+			assertTrue(allFemalesWithMalaria.getMemberIds().contains(7));
+			Cohort allMalesWithFever = (Cohort) row.getColumnValue("FEVER." + OutpatientConsultationReportManager.col17);
+			assertNotNull(allMalesWithFever);
+			assertEquals(1, allMalesWithFever.getSize());
+			Cohort allFemalesWithFever = (Cohort) row.getColumnValue("FEVER." + OutpatientConsultationReportManager.col18);
+			assertNotNull(allFemalesWithFever);
+			assertEquals(0, allFemalesWithFever.getSize());
 			
-			Cohort col5 = (Cohort) row.getColumnValue("DIABETES." + OutpatientConsultationReportManager.col17);
-			assertNotNull(col5);
-			assertEquals(1, col5.getSize());
-			Cohort col6 = (Cohort) row.getColumnValue("DIABETES." + OutpatientConsultationReportManager.col18);
-			assertNotNull(col6);
-			assertEquals(0, col6.getSize());
+			Cohort allMalesWithDiabetes = (Cohort) row
+			        .getColumnValue("DIABETES." + OutpatientConsultationReportManager.col17);
+			assertNotNull(allMalesWithDiabetes);
+			assertEquals(1, allMalesWithDiabetes.getSize());
+			Cohort allFemalesWithDiabetes = (Cohort) row
+			        .getColumnValue("DIABETES." + OutpatientConsultationReportManager.col18);
+			assertNotNull(allFemalesWithDiabetes);
+			assertEquals(0, allFemalesWithDiabetes.getSize());
 			
 			// Referred To column
 			Cohort referredTo1 = (Cohort) row.getColumnValue("MALARIA." + OutpatientConsultationReportManager.col19);
 			assertNotNull(referredTo1);
 			assertEquals(0, referredTo1.getSize());
 			Cohort referredTo2 = (Cohort) row.getColumnValue("MALARIA." + OutpatientConsultationReportManager.col20);
 			assertNotNull(referredTo2);
-			assertEquals(0, referredTo2.getSize());
+			assertEquals(1, referredTo2.getSize());
 			
+			Cohort everyOneWithMalaria = (Cohort) row.getColumnValue("MALARIA." + OutpatientConsultationReportManager.col23);
```

Comment:
`everyOneWithMalaria` → `allWithMalaria`

(but it's "everyone", not "everyOne") ;-)